samples/MiniZinc/data.dzn

```diff
@@ -0,0 +1 @@
+n=3;
```

Comment:
I have added some more realistic code samples for the programming and data languages, taken from the official documents, and copied the MIT license appropriate to those samples to their folders.